addons/binding/org.openhab.binding.fronius/src/main/java/org/openhab/binding/fronius/internal/FroniusHandlerFactory.java

```diff
@@ -31,7 +34,8 @@
 @Component(service = ThingHandlerFactory.class, immediate = true, configurationPid = "binding.fronius", configurationPolicy = ConfigurationPolicy.OPTIONAL)
 public class FroniusHandlerFactory extends BaseThingHandlerFactory {
 
-    private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Collections.singleton(THING_TYPE_INVERTER);
+    private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = ImmutableSet.of(THING_TYPE_INVERTER,
```

Comment:
You do not need the google library here, see for example: https://github.com/eclipse/smarthome/blob/master/extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/HueThingHandlerFactory.java#L42